app/frontend/src/internationalization/trans/chefs/en/en.json

```diff
@@ -325,7 +325,8 @@
     "version": "Version",
     "save": "Save",
     "saving": "Saving",
-    "notSaved": "Not Saved"
+    "notSaved": "Not Saved",
+    "fieldname_error": "You CAN NOT use 'form' keyword as {label} fieldname"
```

Comment:
This is fairly minor, but so far we've never used an underscore in a translation key. For consistency can you change this to `fieldnameError`?

Reply:
@WalterMoar addressed